### PR TITLE
Improve outputFile option work 

### PR DESCRIPTION
**Pull request by hiradimir**
The outputFile option is not work when file not exists.

and some fix.

### CODE REVIEW

**Comment by colingreene28** [comment]
Can we get this in? @palantirtech 

Fails on first run if output file doesn't exists, this means I have to version control the output file, which is not ideal.

File: package.json [hunk 1]
```diff
@@ -27,9 +27,10 @@
     "test": "grunt test"
   },
   "devDependencies": {
-    "grunt-contrib-jshint": "~0.6.0",
+    "grunt": "~0.4.1",
     "grunt-contrib-clean": "~0.4.0",
-    "grunt": "~0.4.1"
+    "grunt-contrib-jshint": "~0.6.0",
+    "tslint": "^2.1.1"
```
Comment (ashwinr) [inline package.json]: why is tslint in devDependencies?

File: tasks/tslint.js [hunk 2]
```diff
@@ -23,7 +23,8 @@ module.exports = function(grunt) {
   grunt.registerMultiTask("tslint", "A linter for TypeScript.", function() {
     var options = this.options({
       formatter: "prose",
-      outputFile: null
+      outputFile: null,
+      appendToOutputFile: false
```
Comment (ashwinr) [inline tasks/tslint.js]: let's rename this to `appendToOutput`

File: tasks/tslint.js [hunk 3]
```diff
@@ -41,11 +42,16 @@ module.exports = function(grunt) {
...
           var outputString = "";
           var outputFile = options.outputFile;
+          var appendToOutputFile = options.appendToOutputFile;
 
           failed += result.failureCount;
 
-          if (outputFile != null) {
-            outputString = grunt.file.read(outputFile);
+          if (outputFile != null && grunt.file.exists(outputFile)) {
+            if(appendToOutputFile){
```
Comment (ashwinr) [inline tasks/tslint.js]: space after `if`, same for the `else` block below

File: package.json (hunk 1 above)
Comment (hiradimir) [inline package.json]: I could not run `grunt test` if its not exist at my env.

File: tasks/tslint.js (hunk 2 above)
Comment (hiradimir) [inline tasks/tslint.js]: ok. I will rename soon.

File: tasks/tslint.js (hunk 3 above)
Comment (hiradimir) [inline tasks/tslint.js]: I fixed it.

**Pull request merged**